app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name.lower()] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_data: list) -> list:
+    name_to_person = {person_data["name"]: Person(
+        name=person_data["name"], age=person_data["age"]
+    ) for person_data in people_data}
```

Comment:
Consider using list comprehension to create the list of `Person` instances. This will make the code more concise and adhere to best practices.